app.js

```diff
@@ -113,6 +113,8 @@ global.Ladders = require(Config.remoteladder ? './ladders-remote.js' : './ladder
 
 global.Users = require('./users.js');
 
+global.Punishments = require('./punishments.js');
```

Comment:
Just requiring it in Users makes it impossible to hotpatch, though... The entire reason we have all these globals is for ease of hotpatching...